src/js/components/Pagination/PageIndex.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { Box } from '../Box';
+import { Button } from '../Button';
+import { Text } from '../Text';
+
+export const PageIndex = ({ page, ...rest }) => {
+  return (
+    <Box as="li">
+      {page === 'more-prev' || page === 'more-next' ? (
```

Comment:
since the ellipsis isn't really a "page", maybe we could add something like a `separator` property. Or, maybe even make that a separate component, like "PageSeparator"?